vignettes/dashboards/index.Rmd

```diff
@@ -259,7 +259,7 @@ ui <- page_sidebar(
 shinyApp(ui, server)
 ```
 
-Filling layout, in general, may not be desirable, especially when there is a lot of output to display on a single page. In this case, consider setting `fillable = FALSE` to prevent growing/shrinking altogether (in this case, outputs then 'fallback' to their default height which is usually 400px).
+Filling layout may not be desirable when there are a lot of outputs to display on a single page. In this case, consider setting `fillable = FALSE` to prevent growing/shrinking altogether. When not using `fillable`, outputs will "fall back" to their default height, which is usually 400px, and scrollbars will be used if the content is too large for its container.
```

Comment:
Thanks, totally agree with direction of this change. I think we can even take another pass:

```suggestion
Filling layout may not be desirable when there are a lot of outputs to display on a single page. In this case, consider setting `fillable = FALSE` to prevent growing/shrinking altogether. When the page layout is not `fillable`, outputs will fall back to their intrinsic height, which is usually 400px. Users will be able to scroll the page if the outputs take up more vertical space than the height of the window.
```

Reply:
Sounds good. I just wanted to add any reference to the word "scroll" because the section header is called "scrolling"